src/main/resources/assets/thaumcraft/models/item/caster_gauntlet.json

```
@@ -0,0 +1,222 @@
+{
+  "ambientocclusion": false,
+  "textures": {
+    "skin": "thaumcraft:entity/armor/caster_advanced"
+  },
+  "elements": [
+    {
+      "from": [
+        6,
+        1.5,
+        6
+      ],
+      "to": [
+        10,
+        7,
+        10
+      ],
+      "faces": {
+        "north": {
+          "uv": [
+            4,
+            1,
+            8,
+            6.5
+          ],
+          "texture": "#skin",
+          "tintindex": 0
+        },
+        "east": {
+          "uv": [
+            4,
+            9,
+            8,
+            14.5
+          ],
+          "texture": "#skin",
+          "tintindex": 0
+        },
+        "south": {
+          "uv": [
+            0,
+            1,
+            4,
+            6.5
+          ],
+          "texture": "#skin",
+          "tintindex": 0
+        },
+        "west": {
+          "uv": [
+            0,
+            9,
+            4,
+            14.5
+          ],
+          "texture": "#skin",
+          "tintindex": 0
+        }
+      }
+    },
+    {
+      "from": [
+        6.25,
+        0,
+        6.25
+      ],
+      "to": [
+        9.75,
+        6,
+        9.75
+      ],
+      "faces": {
+        "north": {
+          "uv": [
+            8,
+            10,
+            12,
+            16
+          ],
+          "texture": "#skin",
+          "tintindex": 0
+        },
+        "east": {
+          "uv": [
+            12,
+            10,
+            16,
+            16
+          ],
+          "texture": "#skin",
+          "tintindex": 0
+        },
+        "south": {
+          "uv": [
+            8,
+            16,
+            12,
+            10
+          ],
+          "texture": "#skin",
+          "tintindex": 0
+        },
+        "west": {
+          "uv": [
+            16,
+            10,
+            12,
+            16
+          ],
+          "texture": "#skin",
+          "tintindex": 0
+        },
+        "down": {
+          "uv": [
+            8,
+            0,
+            12,
+            4
+          ],
+          "texture": "#skin",
+          "tintindex": 0
+        }
+      }
+    },
+    {
+      "from": [
+        6,
+        3,
+        6
+      ],
+      "to": [
+        10,
+        6,
+        10
+      ],
+      "faces": {
+        "up": {
+          "uv": [
+            12,
+            4,
+            16,
+            8
+          ],
+          "texture": "#skin",
+          "tintindex": 0
+        }
+      }
+    }
+  ],
+  "display": {
+    "thirdperson_righthand": {
+      "rotation": [
+        90,
+        90,
+        0
+      ],
+      "translation": [
+        0,
+        -2,
+        8
+      ],
+      "scale": [
+        1.4,
+        1.2,
+        1.4
```

This seems to fix the skin layer clipping while not starting to clip through armor:
```suggestion
        1.3,
        1.192,
        1.3
```
(Same thing for the other gauntlet model variants)